server.go

```diff
@@ -163,21 +163,41 @@ func (s *server) UpdatePin(ctx context.Context, request v0.UpdatePinRequestObjec
 		user = usersResponse.Payload[0]
 	}
 
+	if len(s.cfg.Nuki.SmartLockDevices) == 0 && s.cfg.Nuki.SmartLockDevice == 0 {
```

Comment:
Can we make this check after we load the config and not in the handler?

The server should not even start with such an invalid configuration IMO